### ISSUE TITLE
Uploaded shared text content does not show up in the web ui 

### ISSUE THREAD

**Issue by gerroon** [perkeep/perkeep#1216]
Hi

I am using the android app to send uploads to  my server. I select some text in the browser, then I share that to Perkeep app. it says it uploaded but I do not see anything on the  server side. file uploads  like photos work fine btw.

If this is not supported, please consider supporting such type of uploads.

thanks

**Comment by mpl** [comment]
hmm, I had never thought of that use case, interesting.
What would you expect to happen in that case? that the highlighted text gets saved as a text file?

**Comment by gerroon** [comment]
@mpl 

Yes that was my original intention, that way it can become a way to keep notes. 

I would even like if it saves html files (or scrapes the url) when the actual url  is shared (from the browser) via the share menu to Perkeep.

**Comment by mpl** [comment]
I see. I don't think sharing anything that is not an already existing file is supported at the moment. But I'll see what can be done.

**Comment by gerroon** [comment]
thank for considering.
